demo-modalsheet/src/commonMain/kotlin/ModalBottomSheetDemo.kt

```diff
@@ -38,6 +43,10 @@ fun ModalBottomSheetDemo() {
             detents = listOf(Hidden, Peek, FullyExpanded)
         )
 
+        LaunchedEffect(modalSheetState.targetDetent) {
+            onTargetSheetDetentChange(modalSheetState.targetDetent)
+        }
+
```

Comment:
The Demo is intended for the documentation website. can you reset the change on the demo?

Reply:
Yeah, sure